lib/KeyboardAwareHOC.js

```diff
@@ -321,9 +321,7 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
     }
 
     _resetKeyboardSpace = () => {
-      const keyboardSpace: number = this.props.viewIsInsideTabBar
-        ? _KAM_DEFAULT_TAB_BAR_HEIGHT + this.props.extraScrollHeight || 0
-        : this.props.extraScrollHeight || 0
+      const keyboardSpace: number = this.props.viewIsInsideTabBar ? _KAM_DEFAULT_TAB_BAR_HEIGHT : 0
```

Comment:
Why should we remove `extraScrollHeight`? It would break for people that use that prop.

Reply:
on resetingKeyboardSpace, the default space should be stored

Reply:
The fix in this pull request helped me.

I use ```extraScrollHeight``` and on Android I noticed that after closing the keyboard the page didn't restore its scroll position.

Before fix, link to video:

[![before fix video](http://img.youtube.com/vi/Hh-4nema6wA/0.jpg)](http://www.youtube.com/watch?v=Hh-4nema6wA)

After fix, link to video:

[![after fix video](http://img.youtube.com/vi/XXgPOv3kF2w/0.jpg)](http://www.youtube.com/watch?v=XXgPOv3kF2w)